README.md

```diff
@@ -65,7 +65,7 @@ end
 module ResponseBank
   CACHE_TTL = 30.minutes
   def write_to_backing_cache_store(_env, key, payload, expires_in: CACHE_TTL)
-    cache_store.write(key, payload, raw: true, expires_in: expires_in)
+    cache_store.write(key, payload, raw: true, expires_in: CACHE_TTL)
```

Comment:
What do you think about taking nil and defaulting later?
```ruby
  def write_to_backing_cache_store(_env, key, payload, expires_in: nil)
    cache_store.write(key, payload, raw: true, expires_in: expires_in || CACHE_TTL)
```

As it is, your code will never respect the expires_in if it's passed in.

Reply:
Thanks! updated